File: 9.Revision_OOPS_Fundamental/index_revision_oops_fundamental.py
class User:
    def __init__(self,name,age):
        self.name = name
        self.age = age

    def greet(self):
        print("Hello, I am " + self.name)

    def show_age(self):
        print("My age is " + self.age)

user = User("John", 25)
user.greet();
user.show_age()

File: 9.Revision_OOPS_Fundamental/index_revision_oops_fundamental.js
class User {
  constructor(name, age) {
    this.name = name;
    this.age = age;
  }
  greet() {
    console.log("Hello, I am " + this.name);
  }
  showAge() {
    console.log("My age is " + this.age);
  }
}

const user = new User("John", 25);
user.greet();
user.showAge();